catroid/src/androidTest/java/org/catrobat/catroid/test/devices/mindstorms/ev3/LegoEV3MotorTest.java

```diff
@@ -59,8 +59,8 @@ public void testMotorMoveTest() {
 		int inputSpeed = -70;
 		byte outputField = (byte) 0x01;
 
-		int expectedSpeed = (EV3CommandByte.EV3CommandParamByteCode.PARAM_SHORT_MAX.getByte() & inputSpeed) |
-				EV3CommandByte.EV3CommandParamByteCode.PARAM_SHORT_SIGN_NEGATIVE.getByte();
+		int expectedSpeed = (EV3CommandByte.EV3CommandParamByteCode.PARAM_SHORT_MAX.getByte() & inputSpeed)
+				| EV3CommandByte.EV3CommandParamByteCode.PARAM_SHORT_SIGN_NEGATIVE.getByte();
```

Comment:
Ah, it's not a new feature ^^ Then something like `Extend tracking feature…` or `Add new tracking actions` or `Track additional actions` or whatever you come up with.